docs/03-apis/api-admin/permissions.md

```diff
@@ -116,27 +121,33 @@ default object access permission for a group of a project the request body would
     "forProject":"http://rdfh.ch/projects/0001",
     "forProperty":null,
     "forResourceClass":null,
-    "hasPermissions":[{"additionalInformation":"http://www.knora.org/ontology/knora-admin#ProjectMember","name":"D","permissionCode":7}]
+    "hasPermissions":[
+      {
+        "additionalInformation":"http://www.knora.org/ontology/knora-admin#ProjectMember",
+        "name":"D",
+        "permissionCode":7
+      }
+    ]
 }
 ```
 
 `hasPermissions` contains permission types that must be granted. See [a complete description of object access 
 permission types](../../05-internals/design/api-admin/administration.md#default-object-access-permissions). 
 In summary, each permission should contain followings:
 
-- `additionalInformation`: To whom the permission should be granted: project members, known users, unknown users, etc.
-- `name` : indicates the type of the permission that can be one of the followings.
-   - `RV`: restricted view permission (least privileged)
-   - `V`: view permission
-   - `M` modify permission
-   - `D`: delete permission
-   - `CR`: change rights permission (most privileged)
-- `permissionCode`: The code assigned to a permission indicating its hierarchical level. These codes are as below:
-  - `1`: for restricted view permission (least privileged)
-  - `2`: for view permission
-  - `6`: for modify permission
-  - `7`: for delete permission
-  - `8`: for change rights permission (most privileged)
+  - `additionalInformation`: To whom the permission should be granted: project members, known users, unknown users, etc.
+  - `name` : indicates the type of the permission that can be one of the followings.
+    - `RV`: restricted view permission (least privileged)
+    - `V`: view permission
+    - `M` modify permission
+    - `D`: delete permission
+    - `CR`: change rights permission (most privileged)
+  - `permissionCode`: The code assigned to a permission indicating its hierarchical level. These codes are as below:
+    - `1`: for restricted view permission (least privileged)
+    - `2`: for view permission
+    - `6`: for modify permission
+    - `7`: for delete permission
+    - `8`: for change rights permission (most privileged)
```

Comment:
Why do we need both ways to describe permissions?

Reply:
I have no idea why both are used, I stumbled upon it as well but it is not the scope of this PR.

Reply:
`V1` and `V2`. `V1` use numbers and `V2` uses letters.